tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,15 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel
+
+require('babel-core/register')(require('./babel-config.js')); // #RemoveInProd this line is removed in isopack.js
```

Comment:
So given that @benjamn already has a custom version of babel/register, what's the correct course of action here? Should I switch to that right now, or do something else?

And then what's the final decision on where to put source maps? I thought we said tool startup time might slow down if we use inline maps?